packages/monaco-editor-wrapper/src/codeEditorConfig.ts

```diff
@@ -30,6 +31,7 @@ export class CodeEditorConfig {
 
     // languageclient related configuration
     private useLanguageClient = false;
+    private initializationOptions: any = undefined;
```

Comment:
Is it possible to define this a bit more precisely?
Something like:
```
type InitOptions<T> = { [key: string]: T }
```
`T` can be specific or a union.

Reply:
We could, but that wouldn't really follow the protocol. The [`initializationOptions`](https://microsoft.github.io/language-server-protocol/specifications/lsp/3.17/specification/#initialize) are supposed to be `LSPAny` (so `any` in TypeScript). I don't really see a reason to restrict it any further.

Reply:
Fair point, eslint will complain, please add  `@typescript-eslint/no-explicit-any` where needed.

Reply:
Thanks